File: Java/hydra/src/test/java/uk/co/shastra/hydra/messaging/mocks/DocInfo.java
package uk.co.shastra.hydra.messaging.mocks;

import java.util.Date;

public class DocInfo {
    public String destination;
    public String docId;
    public Date timestamp;
    public String topic;
    
	public DocInfo(String docId, String topic, String destination, Date timestamp) {
		super();
		this.destination = destination;
		this.docId = docId;
		this.timestamp = timestamp;
		this.topic = topic;
	}
    
}

File: DotNet/Tests/Mocks/DocInfo.cs
using System;

namespace Shastra.Hydra.Tests.Mocks
{
    internal class DocInfo
    {
        public string Destination { get; private set; }
        public string DocId { get; private set; }
        public DateTime Timestamp { get; private set; }
        public string Topic { get; private set; }

        public DocInfo(string docId, string topic, string destination, DateTime timestamp)
        {
            DocId = docId;
            Topic = topic;
            Destination = destination;
            Timestamp = timestamp;
        }
    }
}